README.md

```diff
@@ -50,3 +56,9 @@
 
 ## Advance ML-on-Code awareness
 
+### App team activities
+
+- [ ] Speak at at least 1 conference
+- [ ] Write at least 1 blog post
+- [ ] Contribute at least 2 papers on awesome-mloncode
+- [ ] Support the community with source{d} tech stack and MLonCode on Slack
```

Comment:
I again, as I explained to Vadim in his PR, support the community is like breathing or eating, you don't need to put here.

Reply:
If anything, I think it would make sense to create a support KR for every team that is "keep average first response time to 3rd party issues and pull requests to 1 business day", that would be a personal KR for every maintainer.

Reply:
Sorry for confusion, but here I meant mostly our Slack channels. 

👍 on @smola suggestions on setting a KR on some 'time-to-answer' measurement AND notification